src/Transition.js

```diff
@@ -357,6 +361,8 @@ class Transition extends React.Component {
     delete childProps.onExiting
     delete childProps.onExited
 
+    childProps.ref = this.setChildNode
```

Comment:
This will override the ref that is attached to child.

Reply:
And what about user component? Example
```
class Button extends React.Component {}
```

`ref` returned Symbol but not Element